planning/scenario_planning/common/motion_velocity_optimizer/scripts/closest_velocity_checker.py

```diff
@@ -24,11 +24,12 @@
 import copy
 import numpy as np
 
-from autoware_planning_msgs.msg import Path, PathWithLaneId, Trajectory
+from autoware_planning_msgs.msg import Path, PathWithLaneId, Trajectory, VelocityLimit
 from autoware_control_msgs.msg import ControlCommandStamped
-from autoware_vehicle_msgs.msg import VehicleCommand
+from autoware_vehicle_msgs.msg import Engage, VehicleCommand
 from geometry_msgs.msg import Pose, PoseStamped, PoseWithCovarianceStamped, Quaternion, Twist, TwistStamped
-from std_msgs.msg import Bool, Float32, Header, Int32, Float32MultiArray
+from std_msgs.msg import Header
+from autoware_debug.msg import Float32MultiArrayStamped
```

Comment:
`autoware_debug_msgs.msg`?(I think this message is not used in this code, isn't it?)

Reply:
I meant to replace Float32MultiArray with Float32MultiArrayStamped since std_msgs is deprecated.
I have made sure that the code now launches properly.